src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -255,7 +258,10 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     }
     CPU_REGISTER_PASS_COMMON(manager, ov::pass::Validate);
     CPU_REGISTER_PASS_COMMON(manager, ov::pass::RefConvertI64ToI32);
+
+    // todo: remove this conversion line when xxx-105060 is ready and activate conversion at the end of the pipeline
```

Comment:
Are we sure that the following common transformations and LPT would deliver same graph after we move ConvertPrecision down? It doesn't seem obvious from the code. Could you share your thought process?